sdk/appconfiguration/app-configuration/src/models.ts

```diff
@@ -461,7 +461,12 @@ export interface RetryOptions {
 /**
  * Options used when creating a Snapshot.
  */
-export interface CreateSnapshotOptions extends OperationOptions {}
+export interface CreateSnapshotOptions extends OperationOptions {
+  /**
+   * The amount of time to wait (in milliseconds) between subsequent requests relating to the same operation.
+   */
+  updateIntervalInMs?: number;
```

Comment:
`beginCreateSnapshot()` in the generated code takes this as the options bag, unsure how this fell off.

`updateIntervalInMs` makes sense.
`resumeFrom` - I'm not sure about. Should we skip this? Who pauses the operation? service pauses it and we resume it? How is this typically used?

```ts
export interface CreateSnapshotOptionalParams
  extends coreClient.OperationOptions {
  /** Delay to wait until next poll, in milliseconds. */
  updateIntervalInMs?: number;
  /** A serialized poller which can be used to resume an existing paused Long-Running-Operation. */
  resumeFrom?: string;
}


```

Reply:
Maybe it is supported by codegen but I have not tested `resumeFrom` so didn't add it this time.

Reply:
Let's merge the PR anyway since it is unrelated.

@deyaaeldeen if you can shed some light on `resumeFrom`... we can take it in another PR

Reply:
I think the idea is that you serialize the current poller state to a string, then later you can resume by rehydrating the state

Reply:
`resumeFrom` is a string-representation of the poller's state. For instance, the client can decide to serialize the poller state and send it over somewhere else and do polling there instead. I personally don't like it too much because it is an option and you still have to provide required inputs to the method anyway?

Reply:
Also, it is supported in all core-lro's pollers.